core/transport/src/main/java/alluxio/grpc/JobMasterClientServiceGrpc.java

```diff
@@ -479,8 +564,9 @@ protected JobMasterClientServiceFutureStub build(io.grpc.Channel channel,
 
   private static final int METHODID_CANCEL = 0;
   private static final int METHODID_GET_JOB_STATUS = 1;
-  private static final int METHODID_LIST_ALL = 2;
-  private static final int METHODID_RUN = 3;
+  private static final int METHODID_GET_JOB_SERVICE_SUMMARY = 2;
```

Comment:
Does this matter? Should I be putting my method always at the end so that previous methods don't have their values relabeled?

Reply:
I don't think this matters